src/System.IO.FileSystem/tests/Performance/Perf.Directory.cs

```diff
@@ -95,22 +92,27 @@ public void RecursiveCreateDirectory(int depth, int times)
         [Benchmark]
         [InlineData(10, 100)]
         [InlineData(100, 10)]
-        [InlineData(1000, 1)]
         [OuterLoop("Takes a lot of time to finish")]
-        public void RecursiveDeleteDirectory(int depth, int times)
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void RecursiveCreateDirectoryTest(int depth, int times)
         {
-            if (!PlatformDetection.IsWindows && depth == 1000)
-                return;
+            RecursiveCreateDirectory(depth, times);
+        }
+
+        [Benchmark]
+        [OuterLoop("Takes a lot of time to finish")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
@danmosemsft @JeremyKuhne the attribute is not getting respected here. The test stills runs on  unix ? any ideas why is this happening ?